web/views.py

```
@@ -4644,71 +4690,460 @@
+            # For classrooms without courses, find and delete standalone session attendance
+            session = Session.objects.filter(
+                title=f"Class on {today.strftime('%Y-%m-%d')}", start_time__range=(today_start, today_end)
+            ).first()
+
+            if session:
+                SessionAttendance.objects.filter(session=session).delete()
+
+        messages.success(request, "Today's attendance has been reset.")
+        return redirect("classroom_attendance", classroom_id=classroom_id)
+
+    except Exception:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)